### PR TITLE
NIAD-43: API documentation updated

### PR DESCRIPTION
**Pull request by grzegorz-robak**

### CODE REVIEW

File: mhs/outbound/mhsapi.json [hunk 1]
```diff
@@ -1 +1 @@
...
...t payload is base64-encoded or not. This is only required for binary attachments eg images."}, "payload": {"type": "string", "minLength": 1, "maxLength": 5000000, "description": "The attachment, possibly base64-encoded as per is_base64."}, "description": {"type": "string", "minLength": 1, "maxLength": 100, "description": "Description of the attachment"}, "content_type": {"type": "string", "enum": ["text/plain", "text/html", "application/pdf", "text/xml", "application/xml", "text/rtf", "audio/basic", "audio/mpeg", "image/png", "image/gif", "image/jpeg", "image/tiff", "video/mpeg", "application/msword", "application/octet-stream"], "description": "Content type of the attachment"}}, "required": ["content_type", "description", "is_base64", "payload"]}, "RequestBody": {"type": "object", "properties": {"attachments": {"default": [], "maxItems": 98, "description": "Optional attachments to send with the payload. Only for use for interactions that support sending attachments.", "type": "array", "items": {"$ref": "#/components/schemas/Attachment"}}, "payload": {"type": "string", "minLength": 1, "maxLength": 5000000, "description": "HL7 Payload to send to Spine"}}, "required": ["payload"]}}}}
```
Comment (jaroslaw-wojciechowski) [inline mhs/outbound/mhsapi.json]: does it work with pretty printed JSON?
might be easier to review in the future.

Comment (grzegorz-robak) [reply mhs/outbound/mhsapi.json]: it was auto-generated by `pipenv run generate-openapi-docs`

**Review by jaroslaw-wojciechowski — DISMISSED** [review]

File: mhs/outbound/mhsapi.json (hunk 1 above)
Comment (bartek-sarul) [reply mhs/outbound/mhsapi.json]: no switch in the generate-openapi-docs api?

**Review by bartek-sarul — CHANGES_REQUESTED** [review]

File: mhs/outbound/mhsapi.json (hunk 1 above)
Comment (grzegorz-robak) [reply mhs/outbound/mhsapi.json]: it happens that's quite easy to add pretty print - so as requested I've added it

**Review by bartek-sarul — APPROVED** [review]

**Pull request merged**